internal/manifests/manifests.go

```diff
@@ -540,12 +540,12 @@ func (m *Manifests) validateUserSuppliedManifest(ctx context.Context, clusterID
 		if !json.Valid(manifestContent) {
 			return m.prepareAndLogError(ctx, http.StatusBadRequest, errors.Errorf("Manifest content of file %s for cluster ID %s has an illegal JSON format", fileName, string(clusterID)))
 		}
-	} else if strings.HasPrefix(extension, ".patch") && (strings.Contains(fileName, ".yaml.patch") || strings.Contains(fileName, ".yml.patch")) {
+	} else if strings.Contains(fileName, ".yaml.patch") || strings.Contains(fileName, ".yml.patch") {
```

Comment:
just to make sure it's intended,  we accept `myfile.yaml.patch` and `myfile.yaml.patchfoobar`, right?

Reply:
Ideally it should be `myfile.yaml.patch` or `myfile.yaml.patch.foobar`, I'll fix things to account for the extra separator.

Reply:
But yes, any filename containing `.yaml.patch.` or `.yml.patch.` should be permitted